celest/schedule/removal_operators.py

```diff
@@ -0,0 +1,68 @@
+
+
+import random
+
+
+def random_removal(request_handler, number_to_remove):
+
+    if number_to_remove > request_handler.number_of_scheduled_requests:
```

Comment:
This if statement should also hold if equality is met.
```suggestion
    if number_to_remove >= request_handler.number_of_scheduled_requests:
```